networktable.c

```diff
@@ -15,6 +15,7 @@
 
 // Private prototypes
 static char* getMACVendor(const char* hwaddr);
+bool unify_hwaddr(sqlite3 *db);
```

Comment:
If this is not used in this file, does it need to appear here?

Reply:
We enforce prototyping for all functions. Omitting the prototype would cause a compilation error. Typically, the prototype would be included through `routines.h`. Due to the reasons mentioned above (type `sqlite3` is unknown in `routines.h`), I added the prototype here.